app/views/overview/_service-bindings.html

```diff
@@ -8,6 +8,15 @@
     secrets="$ctrl.secrets">
   </overview-service-binding>
   <div ng-if="($ctrl.bindableServiceInstances | size) && ({resource: 'bindings', group: 'servicecatalog.k8s.io'} | canI : 'create')">
-    <a href="" ng-click="$ctrl.createBinding()" role="button">Create Binding</a>
+    <a href="" ng-click="$ctrl.createBinding()" role="button">
+      <span class="pficon pficon-add-circle-o"></span>
```

Comment:
@serenamarie125 added #1913 to cover the issue with icon consistencies on these pages.